app/controllers/dispatch.rb

```diff
@@ -0,0 +1,106 @@
+get '/mission' do
+	# this is turning out to be remarkably similar to the index
+
+
+	# view current missions and upcoming
+	# missions (2 partials)
+	# presented w/ btn to mrk mission start
+
+	unresolved_missions = Mission.where(completed: false).sort_by{|mission| mission.step_off}
```

Comment:
the logic behind finding the unresolved missions should live in the `Mission` model to help keep this app more object oriented. your controller routes should still only accomplish one goal and be single responsibility. for example, we could have something in our `Mission` model that looks like the following:
```ruby
def self.unresolved_missions
 where(completed: false).sort_by{|mission| mission.step_off}
end
```

then we could use this method to find the active and upcoming missions in the model as well. all this logic should live in the model. the controller should only be responsible for handling http requests and grabbing/updating/deleting things from our database.